sotodlib/core/metadata/obsdb.py

```diff
@@ -160,22 +211,33 @@ def update_obs(self, obs_id, data={}, tags=[], commit=True):
             self.
 
         """
+        if (len(self.primary_keys) == 3):
+            wafer_band = self.warn_primary_keys(wafer_band)
+            obs_id = {'obs_id': obs_id, 'wafer_slot': wafer_band[0],
+                      'bandpass': wafer_band[1]}
+        else:
+            obs_id = {'obs_id': obs_id}
```

Comment:
Instead of re-using `obs_id`, call this aggregate var something like `obs_key`.

That's a terminology we could adopt globally.